hafas_event.py

```diff
@@ -54,28 +54,28 @@ def __lt__(self, other):
         return self.realtime < other.realtime
 
     def _clean(self, key):
-        return re.sub(
-            "^(" + REMOVE + "[- ])",
-            "",
-            re.sub(
-                "(\(" + REMOVE + "\))$", "", self.json[key].strip(), flags=re.IGNORECASE
-            ),
-            flags=re.IGNORECASE,
-        ).strip()
+        if key not in self.json:
+            return None
+        else:
```

Comment:
Theoretically, this `else` could be omitted, because of the `return` above, but this is fine as well.